src/libraries/System.Resources.ResourceManager/tests/NeutralResourcesLanguageAttributeTests.cs

```diff
@@ -23,5 +25,25 @@ public static void ConstructorArgumentNull()
         {
             Assert.Throws<ArgumentNullException>(() => new NeutralResourcesLanguageAttribute(null));
         }
+
+        [Theory]
+        [InlineData(typeof(object))] // System.Private.CoreLib,
+        [InlineData(typeof(System.Collections.Stack))] // System.Collections.NonGeneric
+        [InlineData(typeof(System.Collections.Specialized.ListDictionary))] // System.Collections.Specialized
+        [InlineData(typeof(System.Collections.Immutable.IImmutableList<string>))] // System.Collections.Immutable
+        [InlineData(typeof(System.Collections.Concurrent.ConcurrentBag<string>))] // System.Collections.Concurrent
+        [InlineData(typeof(System.Console))] // System.Console
+        [InlineData(typeof(System.IO.Directory))] // System.IO.FileSystem
+        [InlineData(typeof(System.Linq.Enumerable))] // System.Linq
+        [InlineData(typeof(System.Net.Http.HttpClient))] // System.Net.Http
+        [InlineData(typeof(System.Text.RegularExpressions.Regex))] // System.Text.RegularExpressions
+        [InlineData(typeof(System.Threading.Barrier))] // System.Threading
+        public static void TestAttributeExistence(Type type)
+        {
+            CustomAttributeData attribute = type.Assembly.CustomAttributes.FirstOrDefault(a => a.AttributeType.Name == "NeutralResourcesLanguageAttribute");
```

Comment:
Consider making this test case call `ResourceManager.GetNeutralResourcesLanguage` as that was the API reported as being broken in the original issue. https://github.com/dotnet/runtime/issues/13349#issuecomment-536176848

Reply:
I'll try that but wanted to tell GetNeutralResourcesLanguage is protected so I'll still need to use the reflection anyway.

---
In reply to: [453763004](https://github.com/dotnet/runtime/pull/39135#discussion_r453763004) [](ancestors = 453763004)

Reply:
protected static, that's interesting.  You can call it like this:
```c#
public class MyResourceManager : System.Resources.ResourceManager
{
  public static CultureInfo GetNeutralResourcesLanguage(Assembly a)
  {
    return ResourceManager.GetNeutralResourcesLanguage(a);
  }
}
```

Reply:
yes that will work too. I'll try that.

---
In reply to: [453795934](https://github.com/dotnet/runtime/pull/39135#discussion_r453795934) [](ancestors = 453795934)

Reply:
Done!